packages/pdk-nag/scripts/readme-template.mustache

```diff
@@ -0,0 +1,23 @@
+<!-- Auto-generated file - do not modify. -->
```

Comment:
Please add this to a subfolder path as follows: `templates/markdown/awsprototypingrules-readme.mustache`